kernel/src/engine/sync/parquet.rs

```diff
@@ -1,7 +1,9 @@
-use std::fs::File;
-
 use crate::arrow::datatypes::SchemaRef as ArrowSchemaRef;
 use crate::parquet::arrow::arrow_reader::{ArrowReaderMetadata, ParquetRecordBatchReaderBuilder};
+use crate::parquet::arrow::arrow_writer::ArrowWriter;
+use std::fs::File;
+use std::time::SystemTime;
+use url::Url;
```

Comment:
nit: import ordering
```suggestion
use std::fs::File;
use std::time::SystemTime;

use crate::arrow::datatypes::SchemaRef as ArrowSchemaRef;
use crate::parquet::arrow::arrow_reader::{ArrowReaderMetadata, ParquetRecordBatchReaderBuilder};
use crate::parquet::arrow::arrow_writer::ArrowWriter;

use url::Url;
```

Reply:
I assumed that the formatter would take care of that :D